### PR TITLE
GORA-518 Update the policies as per the 4.0.6 aerospike java client

### PR DESCRIPTION
**Pull request by nishadi**
This PR contains changes to update the supported policy configurations as per the 4.0.6 Aerospike java client.
The configurable policies in both [write policy](https://www.aerospike.com/apidocs/java/com/aerospike/client/policy/WritePolicy.html) and [read policy](https://www.aerospike.com/apidocs/java/com/aerospike/client/policy/Policy.html) are made configurable with these changes.

### CODE REVIEW

File: gora-aerospike/src/main/java/org/apache/gora/aerospike/store/AerospikeMappingBuilder.java [hunk 1]
```diff
@@ -16,10 +16,7 @@
  */
 package org.apache.gora.aerospike.store;
 
-import com.aerospike.client.policy.GenerationPolicy;
-import com.aerospike.client.policy.Policy;
-import com.aerospike.client.policy.RecordExistsAction;
-import com.aerospike.client.policy.WritePolicy;
+import com.aerospike.client.policy.*;
```
Comment (djkevincr) [inline gora-aerospike/src/main/java/org/apache/gora/aerospike/store/AerospikeMappingBuilder.java]: Shall we remove this wild card imports?

Comment (nishadi) [inline gora-aerospike/src/main/java/org/apache/gora/aerospike/store/AerospikeMappingBuilder.java]: Removed the wild card import and added separate imports.

File: gora-aerospike/src/main/java/org/apache/gora/aerospike/store/AerospikePolicyConst.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gora.aerospike.store;
+
+public class AerospikePolicyConst {
```
Comment (djkevincr) [inline gora-aerospike/src/main/java/org/apache/gora/aerospike/store/AerospikePolicyConst.java]: Shall we add java doc here?

Comment (nishadi) [inline gora-aerospike/src/main/java/org/apache/gora/aerospike/store/AerospikePolicyConst.java]: Added the java docs.

**Comment by djkevincr** [comment]
@nishadi PR looks good. Will let some time allowing others to take a look.

**Comment by nishadi** [comment]
Thanks :)

**Comment by djkevincr** [comment]
[INFO] Apache Gora ....................................... SUCCESS [1.683s]
[INFO] Apache Gora :: Compiler ........................... SUCCESS [1.866s]
[INFO] Apache Gora :: Compiler-CLI ....................... SUCCESS [1.054s]
[INFO] Apache Gora :: Core ............................... SUCCESS [1:17.569s]
[INFO] Apache Gora :: Accumulo ........................... SUCCESS [1:26.088s]
[INFO] Apache Gora :: Cassandra .......................... SUCCESS [44.335s]
[INFO] Apache Gora :: GoraCI ............................. SUCCESS [8.669s]
[INFO] Apache Gora :: HBase .............................. SUCCESS [2:32.693s]
[INFO] Apache Gora :: Infinispan ......................... SUCCESS [1:17.819s]
[INFO] Apache Gora :: JCache ............................. SUCCESS [59.781s]
[INFO] Apache Gora :: Dynamodb ........................... SUCCESS [3.486s]
[INFO] Apache Gora :: CouchDB ............................ SUCCESS [4.607s]
[INFO] Apache Gora :: Maven Plugin ....................... SUCCESS [2.794s]
[INFO] Apache Gora :: MongoDB ............................ SUCCESS [1:22.538s]
[INFO] Apache Gora :: Solr ............................... SUCCESS [3:01.099s]
[INFO] Apache Gora :: Aerospike .......................... SUCCESS [3.101s]
[INFO] Apache Gora :: Tutorial ........................... SUCCESS [9.868s]
[INFO] Apache Gora :: Sources-Dist ....................... SUCCESS [0.333s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 13:20.072s
[INFO] Finished at: Wed Aug 09 10:01:57 IST 2017
[INFO] Final Memory: 79M/741M

**Comment by djkevincr** [comment]
@nishadi Merged the PR.

**Pull request merged**